library/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -179,7 +184,7 @@ public float getZoom() {
    * Set map position without animation.
    */
   public void setPosition(LngLat lngLat) {
-    mapController.setPosition(lngLat);
+    setPosition(lngLat, 0);
```

Comment:
Nice catch. Filed an issue for us: https://github.com/tangrams/tangram-es/issues/903